game-core/src/main/java/games/strategy/triplea/delegate/battle/MustFightBattle.java

```diff
@@ -718,17 +709,13 @@ public void fight(final IDelegateBridge bridge) {
         dependentUnits,
         attacker,
         defender,
-        isAmphibious(),
+        false,
         getBattleType(),
-        amphibiousLandAttackers);
+        List.of());
     display.listBattleSteps(battleId, stepStrings);
     if (!headless) {
       // take the casualties with least movement first
-      if (isAmphibious()) {
-        CasualtySortingUtil.sortAmphib(attackingUnits, amphibiousLandAttackers);
-      } else {
-        CasualtySortingUtil.sortPreBattle(attackingUnits);
-      }
+      CasualtySortingUtil.sortPreBattle(attackingUnits);
```

Comment:
I can't figure out why these sorts (`sortPreBattle` and `sortAmphib`) happen here.  I tested battles with them removed and things seemed to work the same.  I think that these methods probably existed before the more complex casualty sorter was built and no one got around to removing them.  I worried about removing them as part of this PR but I think they can be removed.

Reply:
The casualty sorter I believe assumes units are provided in an already sorted ordering.